src/All/README.md

```diff
@@ -3,6 +3,13 @@
 ```haskell
 All Boolean
 ```
+<details style="margin:30px">
```

Comment:
Looks like github strips the style attrib.
Bummer.

Reply:
Also this should go in a new constructor "heading" above `Constructor Methods`:

## Constructor
```haskell
All :: a -> All Boolean
```
<details>
  <summary style="margin-bottom:15px">Breakdown</summary>
  <dl>
    <dt><code>value</code></dt><dd>Value to wrap</dd>
    <dt><code>result</code></dt><dd>All wrapping the coerced provided value</dd>
  </dl>
</details>